engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactory.scala

```diff
@@ -72,15 +75,18 @@ object FixedKafkaAvroSourceFactory {
 abstract class BaseKafkaAvroSourceFactory[T: TypeInformation](processObjectDependencies: ProcessObjectDependencies, timestampAssigner: Option[TimestampAssigner[T]])
   extends BaseKafkaSourceFactory(timestampAssigner, TestParsingUtils.newLineSplit, processObjectDependencies) {
 
-  def createKafkaAvroSource(processMetaData: MetaData, topic: String, kafkaAvroSchemaProvider: KafkaAvroSchemaProvider[T]): KafkaSource =
+  def createKafkaAvroSource(processMetaData: MetaData, topic: String, kafkaAvroSchemaProvider: KafkaAvroSchemaProvider[T]): KafkaSource with ReturningType = {
+    //Hack to show error message at creating node
+    val typeDefinition = kafkaAvroSchemaProvider.returnType
```

Comment:
let's use this in returnType?

Reply:
I can do it.. But after this change, we don't have fresh data for latestSchema.. :)